src/utils/get_spoken_word.ts

```diff
@@ -0,0 +1,19 @@
+/** Returns spoken word if it exists on the abbreviation record, otherwise returns the abbreviation */
+export function getSpokenWord(abbreviation: string): string {
+	if (isAbbreviation(abbreviation)) {
+		return abbreviationRecord[abbreviation];
+	}
+
+	return abbreviation;
+}
+
+const abbreviationRecord = {
+	KB: 'kilobyte',
+	MB: 'megabyte',
+	GB: 'gigabyte',
+	AR: 'arweave token'
```

Comment:
Does this risk reading awkwardly when we can't distinguish singular and plural values? Perhaps just saying 'AR' is sufficient since that's the native token name.

Reply:
The difference for this case was between "arweave token input field" and "AR input field"

AR input field sounds just fine in the screen reader, updated in [12a19a5](https://github.com/ardriveapp/price-calculator/pull/24/commits/12a19a53a804e14e915be926986e85861b9bac18)